Urania/Urania.Desktop/MainWindow.xaml.cs

```diff
@@ -77,5 +77,20 @@ public partial class MainWindow {
             WdSwgComboBox.SelectedIndex = -1;
             WdAwgComboBox.SelectedIndex = -1;
         }
+
+        private void HowToUse_Click(object sender, RoutedEventArgs e) {
+            var howToUseText = "Program uwzględnia cztery podstawowe parametry, którymi określamy rozmiar ogniw. " + 
```

Comment:
Oh... god

Reply:
Przenieś to do pliku najlepiej i czytaj z pliku tak jak lubisz i umieć, bo klasa File jest fajna

Reply:
Nie jest i nie lubię.

Reply:
Też czekam aż to zmienisz, żeby dać approve XD